### ISSUE TITLE
Feature Request: "Deferred" Submission Status

### ISSUE THREAD

**Issue by SamsaraTAS** [TASVideos/tasvideos#1265]
"Deferred" or "Holding" or some other name that essentially means "we'll take a look at it later", really.

Basically, this status would serve as a counterpart to Rejected, tagging submissions that are explicitly not being rejected for optimization reasons, i.e submissions that could become acceptable in the future with changes to the site. It would alleviate the stigma of having outright rejections on one's account, something which I've seen members of the community worry about, while also allowing for an extremely easy way to tag and find potentially acceptable runs in the future.

In terms of code, I imagine it'd just be 100% identical to Rejected but with the different name. Status logic would just be Judging Underway > Deferred and Rejected > Deferred.

**Comment by vadosnaprimer** [comment]
Yeah unless it's something that we can't even sync or something sloppy, this decision is "we'll look into it again if it becomes legitimately acceptable". In a way, rejected would become stuff that can't theoretically fit even into playground.

In terms for forum, rejected, canceled, and deferred would go to the same subforum.

**Comment by adelikat** [comment]
With my "quick fix" hat on, I'm wondering if this could just be a rejection reason, which is super easy to add

**Comment by SamsaraTAS** [comment]
As far as I'm aware, the only thing that tracks rejection reasons is the RejectedSubmissions page, which doesn't have sorting and is just a long list of every single rejection. A separate status would allow for workbench filtering to easily bring up a condensed list of just Deferred submissions. Plus, a rejection reason doesn't really address the other issue, which is people being put off by the term "Rejected" itself.

**Comment by meshuggahtas** [comment]
Saw the discussion thread on TASVideos forum, here's my personal opinions:
- Rejected/Rejection is probably the best term to use, used widely in many areas and communities like research, science, school, etc.
- If the staff wants to help submitters avoid extra emotional effect (as any is nearly impossible in an online community) when their submission gets rejected: Life is hard and there will be times where you get a no despite all of your efforts. That's why there are judgement notes with elaborated reasons, that consists of more than a single word explaining what is the problem and how to improve or what to do in order to get accepted.
